gr-pdu/lib/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+# Copyright 2021 Free Software Foundation, Inc.
+#
+# This file is part of GNU Radio
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+########################################################################
+# Setup library
+########################################################################
+add_library(gnuradio-pdu
+    pdu.cc
+    pdu_filter_impl.cc
+    pdu_remove_impl.cc
+    pdu_set_impl.cc
+    pdu_split_impl.cc
+    pdu_to_tagged_stream_impl.cc
+    random_pdu_impl.cc
+    socket_pdu_impl.cc
+    stream_pdu_base.cc
+    tagged_stream_to_pdu_impl.cc
+    tcp_connection.cc
+    tuntap_pdu_impl.cc
+)
+
+#Add Windows DLL resource file if using MSVC
+if(MSVC)
+    include(${CMAKE_SOURCE_DIR}/cmake/Modules/GrVersion.cmake)
+
+    configure_file(
+        ${CMAKE_CURRENT_SOURCE_DIR}/gnuradio-pdu.rc.in
+        ${CMAKE_CURRENT_BINARY_DIR}/gnuradio-pdu.rc
+    @ONLY)
+
+    target_sources(gnuradio-pdu PRIVATE
+        ${CMAKE_CURRENT_BINARY_DIR}/gnuradio-pdu.rc
+    )
+endif(MSVC)
```

Comment:
Is this something new? I haven't seen it in any OOT so far. Or is it specific to this module?

Reply:
I based some of the templated code in this module off gr-trellis and this came from: https://github.com/gnuradio/gnuradio/blob/master/gr-trellis/lib/CMakeLists.txt#L45

I am not a CMake guru so let me know if it should be pulled.

Reply:
This is very MSVC specific. I found [this page](https://docs.microsoft.com/en-us/cpp/windows/resource-files-visual-studio?view=msvc-160). Someone with MSVC experience would be great. As far as I know, we don't have these files in other modules. Thus, I suggest to remove it here as well. It clutters this CMake file and makes the whole process more difficult to understand.

Reply:
All other modules have this, but this was a good catch for windows (which I have not tried)  because I did not include the referenced gnuradio-pdu.rc.in file. I agree someone with MSVC would be awesome to look this over but for now I think doing what other modules have is the right thing.